packages/@react-spectrum/menu/test/MenuTrigger.test.js

```diff
@@ -455,7 +455,7 @@ describe('MenuTrigger', function () {
     it.each`
       Name             | Component      | props
       ${'MenuTrigger'} | ${MenuTrigger} | ${{onOpenChange, closeOnSelect: false}}
-    `('$Name closes menu on item selection via ENTER press even if closeOnSelect=false', function ({Component, props}) {
+    `('$Name doesn\'t closes menu on item selection via ENTER press if closeOnSelect=false', function ({Component, props}) {
```

Comment:
Apparently the previous behavior was intentional on my part, trying to figure out why just in case there was a compelling reason

Reply:
Ah found this: https://github.com/adobe-private/react-spectrum-v3/pull/171#issuecomment-580917542